app/routers/applications.py

```diff
@@ -240,6 +240,12 @@ async def verify_data_field(
         # Update a specific field in the application's `secop_data_verification` attribute.
         payload_dict = {key: value for key, value in payload.dict().items() if value is not None}
         key, value = next(iter(payload_dict.items()), (None, None))
+        if not key:
+            logger.error("Trying to update an empty key")
```

Comment:
This will cause this to be logged in Sentry. Do we want that? As far as I know, it should be impossible to construct a bad request from within the frontend code.